### PR TITLE
Document where to save access controls config files

### PR DESCRIPTION
**Pull request by shawnzhu**
This is based on answers from slack channel on where to save the configuration files for access control. see https://trinodb.slack.com/archives/CGGL0EZ9N/p1611848020001600?thread_ts=1611847702.001500&cid=CGGL0EZ9N

It saved me lots of maintenance effort when scaling worker nodes (by not providing ACL config files on worker nodes).

### CODE REVIEW

File: docs/src/main/sphinx/security/file-system-access-control.rst [hunk 1]
```diff
@@ -23,6 +23,11 @@ The config file is specified in JSON format. It contains rules that define
 which users have access to which resources. The rules are read from top to bottom
 and the first matching rule is applied. If no rule matches, access is denied.
 
+.. note::
+
+    It enforces access control on Trino coordinator, these configuration files for access
```
Comment (sopel39) [inline docs/src/main/sphinx/security/file-system-access-control.rst:28]: change to `Access control is enforced on Trino coordinator, therefore access control configuration files are only required on coordinator node`

**Review by sopel39 — APPROVED** [review]

**Comment by sopel39** [comment]
@mosabua could you ptal?

File: docs/src/main/sphinx/security/file-system-access-control.rst [hunk 2]
```diff
@@ -23,6 +23,11 @@ The config file is specified in JSON format. It contains rules that define
 which users have access to which resources. The rules are read from top to bottom
 and the first matching rule is applied. If no rule matches, access is denied.
 
+.. note::
+
+    Access control is enforced on Trino coordinator, therefore access control
```
Comment (mosabua) [inline docs/src/main/sphinx/security/file-system-access-control.rst:28]: is enforced on the Trino coordinator. Therefore ..

Comment (mosabua) [reply docs/src/main/sphinx/security/file-system-access-control.rst:28]: you could even leave out "Trino"

**Review by mosabua — APPROVED** [review]
Minor change requested but essentially good to go.

File: docs/src/main/sphinx/security/file-system-access-control.rst (hunk 2 above)
Comment (shawnzhu) [reply docs/src/main/sphinx/security/file-system-access-control.rst]: addressed in https://github.com/trinodb/trino/pull/6826/commits/7e91bc39e6de2051b3c9ca314107b7e4bcb3031e

**Review by ebyhr — APPROVED** [review]
Could you squash commits into one?

**Pull request closed without merging**